Formula/vulkan-tools.rb

```diff
@@ -70,7 +71,6 @@ def install
       ]
     end
     system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
-    system "cmake", "--build", "build", "--target", "VulkanTools_generated_source"
```

Comment:
@pthariensflame, why did we need this build target again?

Reply:
- relates to https://github.com/KhronosGroup/Vulkan-Tools/pull/783